buildpipeline/Core-Setup-Signing-Windows-BT.json

```diff
@@ -535,41 +521,49 @@
       "inputs": {}
     },
     {
-      "enabled": false,
-      "continueOnError": false,
+      "enabled": true,
+      "continueOnError": true,
       "alwaysRun": false,
-      "displayName": "Publish Build Artifacts",
+      "displayName": "Copy Publish Artifact: Build Logs",
       "timeoutInMinutes": 0,
       "task": {
-        "id": "bfc8bf76-e7ac-4a8c-9a55-a944a9f632fd",
+        "id": "1d341bb0-2106-458c-8422-d00bcea6512a",
         "versionSpec": "1.*",
         "definitionType": "task"
       },
       "inputs": {
-        "filename": "build.cmd",
-        "arguments": "-Configuration $(BuildConfiguration) -Targets Prepare,Publish $(PB_PortableBuild)",
-        "modifyEnvironment": "false",
-        "workingFolder": "",
-        "failOnStandardError": "false"
+        "CopyRoot": "",
+        "Contents": "**\\*.log",
+        "ArtifactName": "Build Logs",
+        "ArtifactType": "Container",
+        "TargetPath": "\\\\my\\share\\$(Build.DefinitionName)\\$(Build.BuildNumber)"
       }
     },
     {
       "enabled": true,
       "continueOnError": true,
       "alwaysRun": false,
-      "displayName": "Copy Publish Artifact: Build Logs",
+      "displayName": "Cleanup VSTS Agent",
       "timeoutInMinutes": 0,
       "task": {
-        "id": "1d341bb0-2106-458c-8422-d00bcea6512a",
+        "id": "c6c4c611-aa2e-4a33-b606-5eaba2196824",
         "versionSpec": "1.*",
         "definitionType": "task"
       },
       "inputs": {
-        "CopyRoot": "",
-        "Contents": "**\\*.log",
-        "ArtifactName": "Build Logs",
-        "ArtifactType": "Container",
-        "TargetPath": "\\\\my\\share\\$(Build.DefinitionName)\\$(Build.BuildNumber)"
+        "solution": "$(PB_SourcesDirectory)\\Tools\\scripts\\vstsagent\\cleanupagent.proj",
```

Comment:
This is not specifically related to the engine extract / sign / reattach stuff.  But, we've found that VSTS tends to leave builds around and after ~6 weeks, machines will hit out of disk space issues.  So we have this tooling around cleaning up build agents.